### ISSUE TITLE
[MCU] UDS Tx

### ISSUE THREAD

**Issue by Nahrain1** [amosproj/amos2024ss07-updating-flash-boot-loader#44]
## User story
1. As a developer
2. I want to transmit UDS messages from the MCU
3. So that the MCU can communicate with GUI using the UDS protocol

## Acceptance criteria
* Demonstrate the transmission of valid UDS messages to the CAN bus
* All methods are implemented according to Software Architecture
* Hint: uds_comm_spec.c under _Snippets/Testing/src/Communication

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
